moveit_planners/chomp/chomp_motion_planner/src/chomp_trajectory.cpp

```diff
@@ -66,8 +68,15 @@ ChompTrajectory::ChompTrajectory(const moveit::core::RobotModelConstPtr& robot_m
   const moveit::core::JointModelGroup* model_group = robot_model->getJointModelGroup(planning_group_name_);
   num_joints_ = model_group->getActiveJointModels().size();
   init();
+  std::cout << trajectory_ << " complete initialized TRAJECTORY..!!!!!" << std::endl;
 }
 
+/**
+ * this contructor simply copies over the source trajectory into the current trajectory object
```

Comment:
function documentation belongs only in the header file, not cpp

Reply:
sure will remove this.

Reply:
removed.